automation/cleanDockerfile/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fedora:latest
+MAINTAINER Shaked Lokits "shaked.lokits@gmail.com"
+
+
+###############################################################################
```

Comment:
nice affects but make the lines shorter

Reply:
Shaked created this file. I just moved it to a separate folder without touching it at all. But I can make those changes you mentioned

Reply:
Then I'm going back to the question - do we still need this dockerfile?

Reply:
I think yes because we still need to support the old process until we will replace the Jenkinsfile and JJB files to run on Upshift containers